starter/js/atm.js

```diff
@@ -1,43 +1,100 @@
+    // text(something) set the value for certain element
 
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
+  });
 
-        //Else subtract that value from the current amount in the checking account
+    //Checking account withdrawl funtion
+    //On click of the withdrawChecking button
+  $('#withdrawChecking').click(function(){
+    //Get value from the amountChecking input field
+    amountChecking = parseInt($('#amountChecking').val());
+    checkingBalanceTemp = $("#checkingBalance").text(); // get the "$0", you cannot add or substract it directly
+    checkingBalance = parseInt(checkingBalanceTemp.split("").slice(1).join("")); //
+
+    checkingBalance =  checkingBalance- amountChecking;
+    // If that value is greater than the value in the account ignore that action
+    // In other words if this would put the account into a negative balance do not allow it
+    //Else subtract that value from the current amount in the checking account
+   if (checkingBalance>0){
+     $('#checkingBalance').text('$'+ checkingBalance);
+   }else{
+      alert("You don't have enough money");
+   }
+  });
 
     //Savings account deposit function
+   //On click of the depositSavings button
+  $('#depositSavings').click(function(){
+    //Get value from the amountSavings input field
+    //Take that value and add it to the existing value in the savingsBalance div
+    amountSavings = parseInt($('#amountSavings').val());
+    savingsBalanceTemp = $("#savingsBalance").text(); // get the "$0", you cannot add or substract it directly
+    savingsBalance = parseInt(savingsBalanceTemp.split("").slice(1).join("")); //
 
-      //On click of the depositSavings button
+    savingsBalance = amountSavings + savingsBalance;
+    $("#savingsBalance").text('$'+ savingsBalance);
+  });
 
-        //Get value from the amountSavings input field
-
-        //Take that value and add it to the existing value in the savingsBalance div
 
     //Savings account withdraw funtion
-
-      //On click of the withdrawl button
-
+    //On click of the withdrawl button
+  $('#withdrawSavings').click(function(){
+
+    amountSavings = parseInt($('#amountSavings').val());
+    savingsBalanceTemp = $("#savingsBalance").text(); // get the "$0", you cannot add or substract it directly
+    savingsBalance = parseInt(savingsBalanceTemp.split("").slice(1).join(""));
+
+    amountChecking = parseInt($('#amountChecking').val());
+    checkingBalanceTemp = $("#checkingBalance").text(); // get the "$0", you cannot add or substract it directly
+    checkingBalance = parseInt(checkingBalanceTemp.split("").slice(1).join(""));
+
+    if(amountSavings <= savingsBalance) {
+      savingsBalance = savingsBalance - amountSavings;
+      $("#savingsBalance").text('$'+savingsBalance);
+    } else {
+      if((checkingBalance+savingsBalance) >= amountSavings){
```

Comment:
Instead of doing `else` and `if` in two separate statements, you could do the whole thing in one `elseif` addition on the original `if` statement.